org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -382,6 +404,22 @@
             contentTypeId="org.eclipse.wildwebdeveloper.xsl"
             scopeName="source.xml.xsl">
       </scopeNameContentTypeBinding>
+      <grammar
```

Comment:
You don't need to redeclare the grammar as it's already declared above. Just the content-type binding is enough.